src/utils/require-template.js

```diff
@@ -6,24 +6,28 @@ const path = require('path');
 const templateNotFound = 'Error requiring oc-template: "{0}" not found';
 const templateNotValid = 'Error requiring oc-template: "{0}" is not a valid oc-template';
 
-function isValidTemplate(template){
+function isValidTemplate(template, options){
   if (typeof template !== 'object') {
     return false;
   }
 
+  if (options.compiler === true) {
```

Comment:
should this be a if(options.compiler) ? I just wander how it could be === true and also be a function :D